search/tutorials/tutoria_attribute_configuration.md

```diff
@@ -0,0 +1,140 @@
+
+```json
+ "eco-friendly":{
+      "indexable":"false",
+      "searchable":"false",
+      "text":[
+         "Low-impact fabrics",
+         "recycled fabrics",
+         "recycled packaging",
+         "plastic-free packaging",
+         "ethically made"
+      ]
+   }
+```
+
+Let's try to search for a sweater and filter the search results by this field.
+
+To do that you should use a filter **```'(attributes.eco-friendly: ANY("recycled packaging"))'```**
+
+You can find the prepared SearchRequest in the **search/search_attribute_config.py**
+
+Run the sample in a terminal using the command:
+
+```bash
+python search/search_attribute_config.py
+```
+Check the search response printed out to the Terminal.
+As the attribute was not indexed, the filter was not applied to the search resuts, and you see the unfiltered list of products in a response.
+
+## Make the product attribute Indexable
+
+To make the attribute indexable we need to set the "indexable" field to "true". 
+
+Let's do it using the **```update_product()```** method in **search/update_attribute_configuration.py**
+
+Run it in a terminal using the command:
+
+```bash
+python search/update_attribute_configuration.py
+```
+
+## Filter by indexable field
+
+Now, as you have configured the product attribute "eco-friendly" as **indexable**, let's check the Retail API treats it now.
+
+Request the Search Service again with the same request with the filter: ```'(attributes.eco-friendly: ANY("recycled packaging"))'```
+
+```bash
+python search/search_attribute_config.py
+```
+
+As you are searching by the indexable field the filter is applied to the search response and in the result list you will see the requested product.
+
+## Search for non-searchable attribute value
+
+Next, let's request the Retail Search with a query **"Low-impact fabrics"**. This is one of the values of the "eco-friendly" attribute, which is still non-searchable.
+
+The search response should be empty as the Retail Search do not take this attribute into account performing the search.
+
+Open the **search/search_attribute_config.py** and change the value in the field **```search_request.query```**:
+```py
+ search_request.query = "Low-impact fabrics"
+```
+
+Comment out the line ```search_request.filter = '(attributes.eco-friendly: ANY("recycled packaging"))'``` to avoid the narrowing down search results. 
+
+Run the code in a terminal using the command:
+
+```bash
+python search/search_attribute_config.py
+```
+Check the result is empty
+
+## Make the attribute searchable
+
+Next, open the **search/search_attribute_config.py** and change the value of  **attribute.searchable** to "true":
+
+```py
+ attribute.searchable = "true"
+```
+Update the product running the command:
+
+```bash
+python search/update_attribute_configuration.py
+```
+
+## Search for Searchable attribute value
```

Comment:
Search for searchable attribute value

There's no need to capitalize a word here.